test/instrumentation/modules/@aws-sdk/client-dynamodb.test.js

```diff
@@ -0,0 +1,376 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+// Test DynamoDB instrumentation of the '@aws-sdk/client-dynamodb' module.
+//
+// Note that this uses localstack for testing, which mimicks the DynamoDB API but
+// isn't identical. Some known limitations:
+// https://docs.localstack.cloud/user-guide/aws/dynamodb/
+
+const semver = require('semver');
+if (process.env.GITHUB_ACTIONS === 'true' && process.platform === 'win32') {
+  console.log('# SKIP: GH Actions do not support docker services on Windows');
+  process.exit(0);
+}
+if (process.env.ELASTIC_APM_CONTEXT_MANAGER === 'patch') {
+  console.log(
+    '# SKIP @aws-sdk/* instrumentation does not work with contextManager="patch"',
+  );
+  process.exit();
+}
+if (semver.lt(process.version, '14.0.0')) {
+  console.log(
+    `# SKIP @aws-sdk min supported node is v14 (node ${process.version})`,
+  );
+  process.exit();
+}
```

Comment:
nit: As mentioned at https://github.com/elastic/apm-agent-nodejs/pull/3532/files#r1281059147 this *could* be replaced with `versionRanges: { node: '>=14' }` on each test case passed to `runTestFixtures`. Either is fine. Perhaps this one at top of file is cleaner?